pytket/qir/conversion/conversion.py

```diff
@@ -663,16 +665,59 @@ def condition_block() -> None:
                     [
                         pyqir.const(
                             pyqir.Type.double(self.module.module.context),
-                            float(op.params[0]),
+                            float(float(op.params[0]) * math.pi),
                         ),
                         pyqir.const(
                             pyqir.Type.double(self.module.module.context),
-                            float(op.params[1]),
+                            float(float(op.params[1]) * math.pi),
                         ),
                         module.module.qubits[command.qubits[0].index[0]],
                     ],
                 )
 
+            elif op.type == OpType.TK2:
+
+                assert len(command.bits) == 0
+                assert len(command.qubits) == 2
+                assert len(op.params) == 3
+
+                if OpType.TK2 not in self.additional_quantum_gates:
+                    self.additional_quantum_gates[
+                        OpType.TK2
+                    ] = self.module.module.add_external_function(
+                        f"__quantum__qis__rxxyyzz__body",
+                        pyqir.FunctionType(
+                            pyqir.Type.void(self.module.module.context),
+                            [
+                                pyqir.Type.double(self.module.module.context),
+                                pyqir.Type.double(self.module.module.context),
+                                pyqir.Type.double(self.module.module.context),
+                                pyqir.qubit_type(self.module.module.context),
+                                pyqir.qubit_type(self.module.module.context),
+                            ],
+                        ),
+                    )
+
+                self.module.builder.call(  # type: ignore
+                    self.additional_quantum_gates[OpType.TK2],
+                    [
+                        pyqir.const(
+                            pyqir.Type.double(self.module.module.context),
+                            float(float(op.params[0]) * math.pi),
```

Comment:
etc

Reply:
done in @cqc-melf
[remove float conversion](https://github.com/CQCL/pytket-qir/pull/43/commits/e5ae02f70fe8bb6c8705840d0a244a70a83fa78e)
[e5ae02f](https://github.com/CQCL/pytket-qir/pull/43/commits/e5ae02f70fe8bb6c8705840d0a244a70a83fa78e)